Native-Frameworks/FrameworkGridView.swift

```diff
@@ -23,18 +23,22 @@ struct FrameworkGridView: View {
                     .navigationTitle("🍎 Frameworks")
                     .navigationBarItems(trailing: EllipsisButton(isPressed: $isPressed))
                     .fullScreenCover(isPresented: $viewModel.isShowingDetailView) {
-                        FrameworkDetailView(isShowingDetailView: $viewModel.isShowingDetailView, framework: viewModel.selectedFramework ?? MockData.sampleFramework)
+                        FrameworkDetailView(
+                            isShowingDetailView: $viewModel.isShowingDetailView,
+                            framework: viewModel.selectedFramework!
```

Comment:
in `FrameworkGridViewModel` implement  `selectedFramework`  default value so it should never be nil / optional

Reply:
Removed optional and the force unwrapping of `selectedFramework`